static/js/confirm_dialog.js

```diff
@@ -43,12 +58,13 @@ export function launch(conf) {
         "html_heading",
         "html_body",
         "html_yes_button",
+        "loading_spinner",
         "on_click",
         "parent",
     ];
 
     for (const f of conf_fields) {
-        if (!conf[f]) {
+        if (conf[f] === undefined) {
             blueslip.error("programmer omitted " + f);
```

Comment:
I think this belongs in a previous commit, since we introduce `false` as a valid value in the `fade` commit.  Probably best to just make this its own first bug fix commit.